binr/r2r/r2r.c

```diff
@@ -94,6 +152,18 @@ int main(int argc, char **argv) {
 		}
 	}
 
+	if (r2r_dir) {
```

Comment:
You have to fix all paths given as args because if they are relative, they will be broken after cd'ing, making them absolute will probably be enough.

Reply:
then we need to chdir multiple times and chdir-back to the original one after each path assuming the user can pass multiple test files from different working directories

Reply:
I think one run-path for all given paths is enough. But as it is right now, you will be unable to load the tests at all.